### DISCUSSION TITLE
Answered: Ram limiter for Client(s) and Launcher

### DISCUSSION THREAD

**Discussion by teqnik** [WowOpenBox/WowOpenBox#148] [Ideas]
Like isboxer limit ram usage for clients will help computer manage multiple clients
While doing 5 man multibox ram usage for all 5-clients 3-4gb, and it uses nearly all ram. If u can implement a limiter for clients, we can make them to use something like 1gb.
I try to find any app for limiting, but can't find it. If u guys know any app or code/command to limit ram usage let me know pls

Also, a launcher for wow clients like isboxer open 5 clients with own config file for email/accounts selection (I'm using PowerShell command for opening wow clients and capturing)

**Comment by mooreatv** [comment]
> Also, a launcher for wow clients like isboxer open 5 clients with own config file for email/accounts selection (I'm using PowerShell command for opening wow clients and capturing)

So the launcher with different settings for different windows already exist

https://github.com/WowOpenBox/WowOpenBox/wiki/FAQ#i-dont-want-to-use-bnet-i-want-all-my-wow-windows-started-right-away

> Like isboxer limit ram usage for clients will help computer manage multiple clients

There is no such feature that I know of in isboxer nor anywhere really. The ram used is the ram used by the app/game.

What you can probably do is reduce some graphic and other settings to end up with lower foot print per game and/or for all windows but your main.

https://github.com/WowOpenBox/WowOpenBox/wiki/FAQ#my-games-are-lagging
